project1/css/index.css

```diff
@@ -62,3 +62,128 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+
+.container {
+	width: 1200px;
+	border: 2px solid red;
+	margin: 0 auto;
+  }
+  
+  a {
+	color: grey;
+	text-decoration: none;
+	font-size: 15px;
+  }
+  
+  header nav ul li {
+	display: inline-block;
+	margin-left: 75px;
+	margin-top: 40px;
+	color: inherent;
+  }
+  
+  header img {
+	display: block;
+	width: 40%;
+	border: 1px solid green;
+	height: 100px;
+	width: 270px;
+	position: absolute;
+	right: 140px;
+	top: 30px;
+  }
+  
+  h1 {
+	font-size: 80px;
+	margin-top: 100px;
+	text-align: center;
+	margin-right: 350px;
+  }
+  
```

Comment:
Because of the margin size on this H1 element it looks like you're forcing the header image to the next row.

![image](https://user-images.githubusercontent.com/12449668/37049526-53a379c4-212e-11e8-895d-22217b18ce88.png)